backend/tarsy/models/unified_interactions.py

```
@@ -0,0 +1,239 @@
+"""
+Unified interaction models using SQLModel for both runtime and database operations.
+
+This module provides SQLModel classes that serve as both Pydantic models for runtime
+processing AND SQLAlchemy models for database storage, eliminating the need for
+separate runtime/database model hierarchies and manual conversions.
+"""
+
+import uuid
+from typing import Any, Dict, List, Optional
```

Ruff (0.12.2)

`typing.Any` imported but unused

Remove unused import

(F401)